applications/CompressiblePotentialFlowApplication/tests/stochastic_body_fitted_opt/xmc_mlmc_parameters.json

```diff
@@ -0,0 +1,99 @@
+        "costEstimator": "xmc.momentEstimator.MomentEstimator",
+        "costEstimatorInputDictionary": {
+            "indexSetDimension": 0, "order": 1, "updatedPowerSums":"xmc.methodDefs_momentEstimator.updatePowerSums.updatePowerSumsOrder2Dimension0", "centralMomentComputer":"xmc.methodDefs_momentEstimator.computeCentralMoments.centralMomentWrapper", "centralMomentErrorComputer":"xmc.methodDefs_momentEstimator.computeErrorEstimation.centralMomentErrorWrapper"
+        },
+        "qoiEstimator": [
+            "xmc.momentEstimator.MomentEstimator",
+            "xmc.momentEstimator.MultiMomentEstimator",
+            "xmc.momentEstimator.MultiMomentEstimator"
+        ],
+        "qoiEstimatorInputDictionary": [
+            {"indexSetDimension": 1,"order": 1,"centralMomentComputer":"xmc.methodDefs_momentEstimator.computeCentralMoments.centralMomentWrapper","rawMomentComputer":"xmc.methodDefs_momentEstimator.computeCentralMoments.centralMomentWrapper", "rawMomentErrorComputer": "xmc.methodDefs_momentEstimator.computeErrorEstimation.centralMomentErrorWrapper"},
+            {"variableDimension": 12,"order": 1},
+            {"variableDimension": 12,"order": 1}],
+        "sampler": "xmc.sampleGenerator.SampleGenerator"
+    },
+    "monteCarloSamplerInputDictionary": {
+        "assemblersForError": [[2]],
+        "costPredictor": null,
+        "estimatorsForAssembler": [ [[0,[1, true, false]]],[[0,[1, true, false]]],[[0,[1, true, true]]] ],
+        "estimatorsForPredictor": [[ ]],
+        "indexConstructor": "xmc.monteCarloIndex.MonteCarloIndex",
+        "indices": [],
+        "qoiPredictor": [],
+        "initialNumberBatches": 1
+    },
+    "randomGeneratorInputDictionary": {
+        "generator": "xmc.methodDefs_randomGeneratorWrapper.generator.returnUniformAndTwoNormal",
+        "parameters": [0,4294967295,0.3,0.003,-0.035,0.00035]
+    },
+    "samplerInputDictionary": {
+        "randomGenerator": "xmc.randomGeneratorWrapper.RandomGeneratorWrapper",
+        "solverWrapper": "xmc.classDefs_solverWrapper.KratosSolverWrapper.KratosSolverWrapper"
+    },
+    "solverWrapperInputDictionary": {
+        "adaptiveRefinementJumpToFinestLevel": false,
+        "analysisStage": "KratosMultiphysics.CompressiblePotentialFlowApplication.stochastic_potential_flow_response.SimulationScenario",
+        "asynchronous": true,
+        "fakeRandomVariable": [0,0.3,0.0],
+        "mappingOutputQuantities": true,
+        "numberContributionsPerInstance": 1,
+        "outputBatchSize": 1,
+        "projectParametersPath": "primal_parameters.json",
+        "refinementParametersPath": "refinement_parameters.json",
+        "refinementStrategy": "stochastic_adaptive_refinement",
+        "sizeMultiXMomentEstimator": 12
+    },
+    "monoCriteriaInpuctDictionary" :{
+        "statisticalError": {
+            "criteria": "xmc.methodDefs_monoCriterion.criterionFunctions.isLowerThanOrEqualTo",
+            "tolerance": [100.0],
```

Comment:
Same.

Reply:
Here the max iterations are correclty set already: 

https://github.com/KratosMultiphysics/Kratos/blob/c2e956975a2f263e50dc774c571765cbfd43228a/applications/CompressiblePotentialFlowApplication/tests/stochastic_body_fitted_opt/xmc_mlmc_parameters.json#L77

Reply:
I am referring to the tolerance, that is very high. I would place here the tolerance that you are actually satisfying.

Reply:
I dont really see the point, the tolerance is kinda meaningless here, the main interest is just to go through the code once for testing purposes (which is anyway controlled by the maxIter key). I changed it back in the other json to reduce the changes since the other value was already in place.

Reply:
As you prefer, I was commenting only because if an external user sees this he/she may wonder why it is so high.